steps/metabase/base/metabase.go

```diff
@@ -0,0 +1,88 @@
+package base
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	gabs "github.com/Jeffail/gabs/v2"
+	"github.com/stackpulse/steps-sdk-go/env"
+)
+
+type MetabaseCommandArgs struct {
+	Username  string `env:"USERNAME,required"`
+	Password  string `env:"PASSWORD,required"`
+	Host      string `env:"HOST,required" envDefault:"metabase.data.svc.cluster.local"`
+	Token     string `env:"TOKEN," envDefault:""`
+	Recipient string `env:"RECIPIENT"`
+	UserId    string
+}
+
+type Output struct {
+	User    string
+	Message string
+}
+
+type MetabaseCommand struct {
+	MetabaseCommandArgs
+}
+
+func NewMetabaseCommand(args *MetabaseCommandArgs) (*MetabaseCommand, error) {
+	if err := env.Parse(args); err != nil {
+		return nil, fmt.Errorf("parse args: %w", err)
+	}
+	return &MetabaseCommand{MetabaseCommandArgs: *args}, nil
+}
+
+func (mb *MetabaseCommand) GetUserId() (*MetabaseCommand, error) {
```

Comment:
no need return MetabaseCommand
```suggestion
func (mb *MetabaseCommand) GetUserId() error {
```